.github/workflows/backend-dev-cd.yml

```
@@ -75,12 +72,12 @@ jobs:
-          DEV_DB_RO=${{ secrets.DEV_DB_RO }}
-          DEV_DB_RO_PASSWORD=${{ secrets.DEV_DB_RO_PASSWORD }}
-          DEV_DB_RW=${{ secrets.DEV_DB_RW }}
-          DEV_DB_RW_PASSWORD=${{ secrets.DEV_DB_RW_PASSWORD }}
-          DEV_DB_MIG=${{ secrets.DEV_DB_MIG }}
-          DEV_DB_MIG_PASSWORD=${{ secrets.DEV_DB_MIG_PASSWORD }}
+          DEV_DATASOURCE_URL=${{ secrets.DEV_DATASOURCE_URL }}
+          DEV_DATASOURCE_RW_USERNAME=${{ secrets.DEV_DATASOURCE_RW_USERNAME }}
+          DEV_DATASOURCE_RW_PASSWORD=${{ secrets.DEV_DATASOURCE_RW_PASSWORD }}
+          DEV_DATASOURCE_MIGRATION_USERNAME=${{ secrets.DEV_DATASOURCE_MIGRATION_USERNAME }}
+          DEV_DATASOURCE_MIGRATION_PASSWORD=${{ secrets.DEV_DATASOURCE_MIGRATION_PASSWORD }}
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)